be/src/exec/csv_scanner.cpp

```diff
@@ -39,6 +39,26 @@ std::string string_2_asc(const std::string& input) {
     return output;
 }
 
+static std::string make_column_count_not_matched_error_message(int expected_count, int actual_count,
+                                                               CSVParseOptions& parse_options) {
+    std::stringstream error_msg;
+    error_msg << "Value count does not match column count: "
+              << "expected = " << expected_count << ", actual = " << actual_count << ". "
+              << "Column delimiter: " << string_2_asc(parse_options.column_delimiter) << ", "
```

Comment:
Print 'Column separator' to match the parameters `column_separator` and `row_delimiter`.

Reply:
And, can you print the `column_separator` using `\t`, `\n`, and hexadecimal code like `0x0D`, It's more readable than decimal code `9`, `10`.

Reply:
fixed.